invisible_flow/entities/data_policeunit.py

```diff
@@ -0,0 +1,8 @@
+from attr import dataclass
+
+
+@dataclass
+class PoliceUnit:
```

Comment:
Can we make this file name `data_police_unit`?

Reply:
We could do this, but it does not follow the convention already established by allegation, complainant, and investigator